packages/sdk/src/apiAccount.ts

```diff
@@ -813,4 +813,43 @@ export class ApiAccount<T = void> extends WithAccountHistory implements ISubmitE
       return '0';
     }
   }
+
+  public createMST(accounts: string[], threshold: number, name: string): string {
+    const result = keyring.addMultisig(accounts, threshold, { name });
+    const addressMST = this.formatAddress(result.pair.address);
+    keyring.saveAddress(addressMST, {
+      name,
+      isMultisig: true,
+      whenCreated: Date.now(),
+      threshold,
+      who: accounts,
+    });
+    // In default account set MST Address account
+    this.accountStorage?.set('MSTAddress', addressMST);
+
+    return addressMST;
+  }
+
+  public getMstAccount(address: string): KeyringAddress | undefined {
+    const multisigAccounts = keyring.getAddresses().filter(({ meta }) => meta.isMultisig);
+    const multisigAccount = multisigAccounts.find((account) => {
+      const accountAddress = this.formatAddress(account.address, false);
+      const targetAddress = this.formatAddress(address, false);
+      return accountAddress === targetAddress;
+    });
+    return multisigAccount;
+  }
+
+  public updateMultisigName(newName: string): void {
+    const addressMST = this.formatAddress(this.account?.pair?.address) ?? '';
```

Comment:
Shouldn't here be:
`this.accountStorage?.get('MSTAddress')`

Because in `createMST` method you saved MST acc address to accountStorage?